crowbar_framework/app/models/api/crowbar.rb

```diff
@@ -113,7 +113,7 @@ def compute_status
           )
         end
         nova = NodeObject.find("roles:nova-controller").first
-        ret[:no_live_migration] = true unless nova["nova"]["use_migration"]
```

Comment:
If there is no nova, then there is no error and upgrade will just be faster as we do not need to do the live-migration. I think it's good enough to have checked like the one in PR